src/data-browser/graphs/quarterly/SectionGraphs.jsx

```diff
@@ -76,57 +92,104 @@ export const SectionGraphs = ({
   })
 
   const handleGraphSelection = useCallback(
-    event => {
-      dispatch(graphs.setConfig(SELECTED_GRAPH, rawGraphList.find(opt => opt.value == event.value)))
+    (event) => {
+      dispatch(
+        graphs.setConfig(
+          SELECTED_GRAPH,
+          rawGraphList.find((opt) => opt.value == event.value)
+        )
+      )
       fetchSingleGraph(event.value) // value = endpoint for single graph (i.e) -> /applications
     },
     [rawGraphList, fetchSingleGraph, dispatch]
   )
 
-
   // Update the URL when query parameters change
   useEffect(() => {
     if (!periodLow || !periodHigh || !selectedGraph || !seriesForURL) return
 
-    props.history.push({
-      pathname: `${BaseURLQuarterly}/${selectedGraph.value}`,
-      search: `?periodLow=${periodLow.value}&periodHigh=${periodHigh.value}&visibleSeries=${seriesForURL}`,
-    })
+    // Function used to reduce re-writing same dispatch method to redux store
+    const dispatchGraphURL = (
+      selectedGraph,
+      periodLow,
+      periodHigh,
+      seriesForURL
+    ) => {
+      dispatch(
+        graphs.setConfig(
+          GRAPH_URL,
+          `${BaseURLQuarterly}/${selectedGraph}/?periodLow=${periodLow}&periodHigh=${periodHigh}&visibleSeries=${seriesForURL}`
+        )
+      )
+    }
+
+    // Allows direct linking to /info/filer or /info/faq and doesn't trigger a url update
+    // Additonally it stores the graph url and is used when user clicks the Graphs tab
+    if (props.history.location.pathname.includes("/info/filer")) {
+      props.history.push({
+        pathname: `${BaseURLQuarterly}/info/filer`,
+      })
+      dispatchGraphURL(
```

Comment:
`dispatchGraphURL` call is repeated for every condition which means it's not conditional.  Maybe have one call after the `if-else` block since it is currently called *after* the `history.push` in every case?

Reply:
Good catch, I can't remember why I added the dispatch method to every `if-else` block. I updated the changed to dispatch outside all the `if-else`.